app/assets/javascripts/modules/moj.AsyncGA.js

```diff
@@ -56,6 +88,17 @@
       } else {
         ga('send', 'pageview');
       }
+    },
+    hideCookieBanner: function() {
+      document.getElementById('cookie-message').style.display = 'none';
+    },
+    removeCookie: function(name) {
+      document.cookie = name + '=; Max-Age=0';
+    },
+    cookieOneYearExpiration: function() {
+      var date = new Date();
+      date.setTime(+date + 365 * 86400000);
```

Comment:
I think this could be easier to follow if written as:

```js
var date = new Date();
date.setFullYear(date.getFullYear() + 1);
```

The use of 'year' methods and the `+1` on the value feels easier to parse, rather than trying to remember what the number `86400000` means in this context.

This is just a personal preference, though, so absolutely no need to change what's already here. You'd already made the intent of this method quite clear through its name.

Reply:
Wow, this definitely looks allot better! I've now made this change.